scripts/terminalserver/terminalserver.jl

```diff
@@ -150,6 +150,11 @@ end
             source_code = payload_as_string[end_second_line_pos+1:end]
 
             hideprompt() do
+                if isdefined(Main, :Revise)
```

Comment:
I don't have any understanding of how Revise works - is it possible JULIA_REVISE is set without it being turned on in the `julia.useRevise` vscode setting? If so would we want a more robust check that Main.Revise points to the package?

Reply:
Ah, good point, `Main.Revise` could be something other than the Revise.jl package. I've added a couple more tests. Still not a 100%, but maybe this is robust enough for now?

And yes, Revise.jl can for example be loaded in the startup script of the user, even if our setting is to not load it. That is maybe something we should also think about: maybe we should have our own startup script (like IJulia does) and not run the default startup script in our REPL... But we can decide that later, I'd say.